frontend/components/ChatInput.tsx

```
@@ -0,0 +1,180 @@
+                  <div className="text-xs font-semibold text-gray-500 mb-2">All Personas</div>
+                  <div className="text-xs text-gray-500 mb-2">Mention everyone in this scene</div>
+                  {personas.map((persona) => (
+                    <div
+                      key={persona.id}
+                      className="flex items-center gap-2 p-2 hover:bg-gray-50 rounded cursor-pointer"
+                      onClick={() => {
+                        const mentionId = persona.name.toLowerCase().replace(/\s+/g, '_');
+                        setInput(input.replace(/@[^@]*$/, `@${mentionId} `));
+                        setShowMentionDropdown(false);
+                      }}
+                    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)